pkg/arvo/sys/vane/gall.hoon

```diff
@@ -1849,6 +1854,7 @@
       ~/  %ap-poke
       |=  =cage
       ^+  ap-core
+      ~>  %spin.[(crip "on-poke/{<agent-name>}")]
```

Comment:
Should this maybe put the mark in there also?

Reply:
Similarly, the wire for `+on-agent` and `+on-arvo`?

Reply:
Talked to @joemfb and @frodwith about this in CCRS. They wanted to keep this hint a bit higher level. I think we don't put this information in it at this time. If we think its needed for future info we can always add it again